sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -20,6 +22,7 @@ public partial class LogsQueryClient
     public partial class LogsQueryClientOptions : Azure.Core.ClientOptions
     {
         public LogsQueryClientOptions(Azure.Monitor.Query.LogsQueryClientOptions.ServiceVersion version = Azure.Monitor.Query.LogsQueryClientOptions.ServiceVersion.V1) { }
+        public string AuthenticationScope { get { throw null; } set { } }
```

Comment:
Is this auto-generated code? I'm kind of surprised to see a throw inside of the property here.

Reply:
It's the API export

Reply:
Yeah, that's the exported API. The API exporter produces something that can be compiled so it replaces method bodies with `throw null;`.

Reply:
Ah, that is interesting. Our equivalent of this file is just a api.md file :)